contracts/ssnlist.scilla

```diff
@@ -14,6 +14,7 @@ let bool_active = True
 let bool_inactive = False
 let uint128_10_power_7 = Uint128 10000000
 let empty_tag = ""
+let add_funds = "AddFunds"
```

Comment:
may be want to consider renaming `add_funds` to either `add_funds_tag` or `tag_addfunds`?

Reply:
Oh `add_funds_tag` will be nicer!